optimum/neuron/models/training/transformations_utils.py

```diff
@@ -88,6 +88,7 @@ def adapt_state_dict(
         named_parameters: Dict[str, torch.nn.Parameter],
         orig_state_dict: Dict[str, torch.Tensor],
         inplace: bool = False,
+        upstanding_sharded_params: Dict[str, torch.Tensor] = {},
```

Comment:
```suggestion
        upstanding_sharded_params: Optional[Dict[str, torch.Tensor]] = None,
```

Then do:
```
if upstanding_sharded_params is None:
    upstanding_sharded_params = {}
```

Because mutable defaults can be problematic, especially when there are multiple calls of the function.

Reply:
I will make this mandatory then.